tests/bearlib/abstractions/LinterTest.py

```diff
@@ -727,7 +732,8 @@ def test_repr(self):
             '<ManualProcessingTestLinter linter object \\(wrapping ' +
             re.escape(repr(sys.executable)) + '\\) at 0x[a-fA-F0-9]+>')
 
-    @skipIf(platform.system() == 'Windows',
+    @skipIf(platform.system() == 'Windows' and
+            not is_executable_exists('pwd'),
             '`pwd` does not exist in Windows-cmd and `cd` is a built-in '
             'command which fails the executable-existence check from @linter.')
```

Comment:
Finally completely remove the `@skipIf` and set `linter` executable to `cmd` on Windows